src/sections/accordion.scss

```diff
@@ -1,3 +1,111 @@
 .accordion-cont {
-    height: 15rem;
+    margin-top: 1rem;
+    width: 100%;
+    height: auto;
+    position: relative;
+    border: 8px solid black;
```

Comment:
Maybe it is good to make border white like our movies poster should have according to design because when comes background image - which is dark - border won't be visible.

Reply:
It is okay to change it after background comes.